### DISCUSSION TITLE
I Find a Sound Issue in my current running video when i started obs in Background

### DISCUSSION THREAD

**Discussion by Raafey-Khan** [obsproject/obs-studio#10224] [Development Questions]
So i was just watching my coding lectures on chrome, and i thought to open OBS and the moment obs opened on my macbook, the audio which is coming from my coding lecture, that got changed into muffed sound ,and dimmed sound like that, i don't know why its occuring, but in case if any developer is seeing my message please fix the sound issue that causing trouble . @

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
